django/contrib/admin/templates/admin/base.html

```diff
@@ -6,21 +6,23 @@
 <link rel="stylesheet" href="{% block stylesheet %}{% static "admin/css/base.css" %}{% endblock %}">
 {% block dark-mode-vars %}
   <link rel="stylesheet" href="{% static "admin/css/dark_mode.css" %}">
-  <script src="{% static "admin/js/theme.js" %}"></script>
+  <script src="{% static "admin/js/theme.js" %}" defer></script>
```

Comment:
1. I don't see how this change is related to the ticket?
2. The theme script applies the theme from the `localStorage`, so I suspect if you don't have the default value set, you may get brief flickering between the loading of the script and it finally executing. Or am I misunderstanding `defer`?

Reply:
Removed defer from theme.js